metadata.json

```diff
@@ -31,7 +31,9 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "10"
+        "10",
+        "11",
+        "12"
```

Comment:
Debian 12 is not available yet :smile:

Reply:
What I mean is that when Debian 12 is released, we will add config to add support for running acceptance tests on it, and only then we will be able to see if for some reason the module works or not with this version of Debian.  So it would be more consistent to not yet add Debian 12, and open a PR to add support for it when everything is ready for testing it.

Reply:
Well the test fail anyways due to some strange behavior.

Reply:
Looks like the update of beaker break CI…

beaker 4.39.0 was passing (CI in #12), beaker 5.1.0 is broken.
Beaker 4 had a dependency on beaker-pe, which seems related to the error we see :suspect:.

Reply:
Yeah - I thought about something like that. Didn't have time to check it.